api/tests/routes/pro/post_offer_test.py

```diff
@@ -144,7 +150,41 @@ def test_create_valid_educational_offer_with_new_route(self, app):
             "visualDisabilityCompliant": False,
         }
         client = TestClient(app.test_client()).with_session_auth("user@example.com")
-        response = client.post("/offers/educational", json=data)
+        with requests_mock.Mocker() as request_mock:
```

Comment:
Est-ce qu'on ne ferait pas un test d'un cas non passant?
Si non ici, au moins au niveau du use case `api.create_educational_offer`? Qu'on s'assure que l'offre n'est pas créée (i.e. pas présente en base) en cas de retour not found d'adage?
J'avoue que je serais chaud pour qu'on ajoute ça :)